src/main/app-updater.ts

```diff
@@ -41,8 +41,14 @@ function handleAutoUpdateBackChannel(event: Electron.IpcMainEvent, ...[arg]: Upd
     if (arg.now) {
       logger.info(`${AutoUpdateLogPrefix}: User chose to update now`);
       setImmediate(() => {
-        app.removeAllListeners("window-all-closed");
-        WindowManager.getInstance().destroy();
```

Comment:
So no need to do this for any OS?

Reply:
Nope, that was added by me in an attempt to debug this.